docs/LINUX.md

```diff
@@ -0,0 +1,121 @@
+# LogDNA Agent on Linux distributions
+
+The agent supports 64-bit (`x86_64`) Linux distributions and provides rpm and deb packages.
+
+## Installing
+
+### On Debian-based distributions
+
+```shell script
+echo "deb https://assets.logdna.com stable main" | sudo tee /etc/apt/sources.list.d/logdna.list
+wget -qO - https://assets.logdna.com/logdna.gpg | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install logdna-agent
+```
+
+### On RPM-based distributions
+
+```shell script
+sudo rpm --import https://assets.logdna.com/logdna.gpg
+echo "[logdna]
+name=LogDNA packages
+baseurl=https://assets.logdna.com/el6/
+enabled=1
+gpgcheck=1
+gpgkey=https://assets.logdna.com/logdna.gpg" | sudo tee /etc/yum.repos.d/logdna.repo
+sudo yum -y install logdna-agent
+```
+
+# Usage
+
+You can start sending logs by setting the [ingestion key][ingestion-key]:
+
+```shell script
+logdna-agent -k <YOUR INGESTION KEY>
+```
+
+The agent expose short argument abbreviations for commonly used options (`-k`, `-t`, `-d` and `-c`).
+
```

Comment:
Suggest we say "The LogDNA Agent exposes short..." (need the "s" in "exposes" for sure.)